src/main/java/ch/wisv/areafiftylan/model/User.java

```diff
@@ -139,4 +139,33 @@ public void setAccountNonLocked(boolean accountNonLocked) {
     public void setEnabled(boolean enabled) {
         this.enabled = enabled;
     }
+
+    @Override
```

Comment:
User can be annotated with `@EqualsAndHashCode` to generate these automatically.